cnf-tests/bin/.gitignore

```diff
@@ -0,0 +1,4 @@
+# Ignore everything in this directory
+*
+# Except this file
+!.gitignore
```

Comment:
I think adding `bin` to the root .gitignore is easier?

Reply:
I think it won't work because git won't let me commit empty folders. 
See https://stackoverflow.com/questions/115983/how-can-i-add-an-empty-directory-to-a-git-repository

Reply:
do you need to commit it? :thinking:

Reply:
I think so, otherwise the bin folder won't persist if you checkout on a new machine.
Not sure I understand what your doubt is...

Reply:
I just don't understand why you want to persist the folder.
I would just `mkdir -p` it when needed and ignore it completely.
But maybe it's just a matter of taste. Will lgtm and hold, feel free to remove the hold when you prefer to keep it as it is :slightly_smiling_face: 

/lgtm
/hold

Reply:
I'm fine with either of them.

I changed it as per @SchSeba comment: https://github.com/openshift-kni/cnf-features-deploy/pull/138#discussion_r404706095

Reply:
Uhm, now I realize I mis-interpreted it. Changing.